bitnami/prometheus-operator/templates/prometheus/thanos-service.yaml

```diff
@@ -29,6 +29,10 @@ spec:
       {{- if and .Values.prometheus.thanos.service.nodePort (or (eq .Values.prometheus.thanos.service.type "NodePort") (eq .Values.prometheus.thanos.service.type "LoadBalancer")) }}
       nodePort: {{ .Values.prometheus.thanos.service.nodePort }}
       {{- end }}
+    - name: http
+      port: 10902
+      targetPort: http
+      protocol: TCP
```

Comment:
I would do it customizable as grpc port is with `.Values.prometheus.thanos.service.port`. In fact, you could refactor this values:

```yaml
prometheus:
  thanos:
    service:
      ports:
        grpc: 10901
        http: 10902
```

then use them
```
.Values.prometheus.thanos.service.ports.grpc
.Values.prometheus.thanos.service.ports.http
```

Reply:
@dani8art that would require a migration path for existing users (and as a result a major bump in Chart version). That's mainly the reason why I did not do it like this.

Reply:
Yes you are right, and what about the `Values.prometheus.thanos.service.extraPorts` approach? please take a look at this https://github.com/bitnami/charts/blob/b0c0732b5ef07d31a84cb64d2770b34fee6ecc57/bitnami/discourse/templates/service.yaml#L36

Reply:
then you could set it by default in values:

```yaml
prometheus:
  thanos:
    service:
      extraPorts:
        - name: http
          port: 10902
          targetPort: http
          protocol: TCP
```

Reply:
@dani8art that sounds like a very good suggestion. Thanks

Reply:
It's done